lang/fr-on.js

```diff
@@ -0,0 +1,101 @@
+	"components.list-item-drag-handle.default": "Réordonner l'action de l'élément pour {name}",
+	"components.list-item-drag-handle.keyboard": "Réorganiser les éléments, position actuelle {currentPosition} de {size}. Pour déplacer cet élément, utilisez les flèches vers le haut et vers le bas.",
+	"components.list-item-tooltip.title": "Navigation au clavier pour les listes :",
+	"components.list-item-tooltip.enter-key": "Entrée",
+	"components.list-item-tooltip.enter-desc": "Activer l'option de mise au point.",
+	"components.list-item-tooltip.up-down-key": "Haut/bas",
+	"components.list-item-tooltip.up-down-desc": "Déplacer la priorité entre les éléments d'une liste.",
+	"components.list-item-tooltip.left-right-key": "Gauche/droite",
+	"components.list-item-tooltip.left-right-desc": "Déplacer la priorité dans l’élément courant.",
+	"components.list-item-tooltip.page-up-down-key": "Page haut/bas",
+	"components.list-item-tooltip.page-up-down-desc": "Déplacer la priorité cinq éléments à la fois.",
+	"components.list-item-drag-handle-tooltip.title": "Commandes du clavier pour la réorganisation :",
+	"components.list-item-drag-handle-tooltip.enter-key": "Entrée",
+	"components.list-item-drag-handle-tooltip.enter-desc": "Basculer en mode réorganiser le clavier.",
+	"components.list-item-drag-handle-tooltip.up-down-key": "Haut/bas",
+	"components.list-item-drag-handle-tooltip.up-down-desc": "Déplacer l'élément vers le haut ou vers le bas dans la liste.",
+	"components.list-item-drag-handle-tooltip.left-right-key": "Gauche/droite",
+	"components.list-item-drag-handle-tooltip.left-right-desc": "Changer le niveau d'imbrication.",
+	"components.menu-item-return.return": "Retour au menu précédent.",
+	"components.menu-item-return.returnCurrentlyShowing": "Retour au menu précédent. Vous voyez actuellement {menuTitle}.",
+	"components.meter-mixin.commaSeperatedAria": "{term1}, {term2}",
+	"components.meter-mixin.fraction": "{x}∕{y}",
+	"components.meter-mixin.progressIndicator": "Indicateur de progrès",
+	"components.more-less.less": "moins",
+	"components.more-less.more": "plus",
+	"components.overflow-group.moreActions": "Plus d'actions",
+	"components.selection.action-hint": "Sélectionner un élément pour exécuter cette action.",
+	"components.selection.select-all": "Tout sélectionner",
+	"components.selection.select-all-items": "Sélectionner tous les {count} éléments",
+	"components.selection.selected": "{count} sélectionné(s)",
+	"components.switch.visibility": "Visibilité",
+	"components.tabs.next": "Défilement avant",
+	"components.tabs.previous": "Défilement arrière",
+	"templates.primary-secondary.adjustableSplitView": "Vue partagée réglable",
+	"templates.primary-secondary.keyboardHorizontal": "Utiliser la flèche vers la gauche ou vers la droite pour régler la taille des volets d'affichage",
+	"templates.primary-secondary.keyboardVertical": "Flèche vers le haut ou vers le bas pour régler la taille des volets d'affichage"
+};
```

Comment:
I think I'm ready to give in and just allow the `fr-ON` translations through. If these end up deviating from `fr.js` then that could be bad when/if we ever swap out `fr-ON` for OSLO, since core doesn't plan on supporting OSLO.